proto/monitor_service.proto

```diff
@@ -54,25 +54,29 @@ message AnalyzeHeapResponse {
 // Back pressure
 message GetBackPressureRequest {}
 
-message BackPressureInfo {
-  uint32 actor_count = 3;
-  // Sum of output blocking duration of all actors
-  double value = 4;
+message ChannelStats {
+  uint32 actor_count = 1 [json_name = "n"];
+  // Sum of output blocking duration of all actors in nanoseconds
+  double blocking_duration = 2 [json_name = "b"];
+  // Sum of input row count of all actors
+  uint64 input_row_count = 3 [json_name = "i"];
+  // Sum of output row count of all actors
+  uint64 output_row_count = 4 [json_name = "o"];
 }
 
 message FragmentStats {
-  uint32 actor_count = 2;
-  uint64 current_epoch = 3;
+  uint32 actor_count = 2 [json_name = "n"];
+  uint64 current_epoch = 3 [json_name = "e"];
 }
 
 message RelationStats {
-  uint32 actor_count = 2;
-  uint64 current_epoch = 3;
+  uint32 actor_count = 2 [json_name = "n"];
+  uint64 current_epoch = 3 [json_name = "e"];
 }
 
 message GetBackPressureResponse {
```

Comment:
Maybe better to rename this API, and add some comments, since it's not only for backpressure.